db/data/20220720142258_change_newflow_social_to_facebook_google.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class ChangeNewflowSocialToFacebookGoogle < ActiveRecord::Migration[5.2]
+  def up
+    Authentication.where(provider: 'facebooknewflow').each do |auth|
+      auth.provider = 'facebook'
+      auth.save
+    end
+
+    Authentication.where(provider: 'googlenewflow').each do |auth|
+      auth.provider = 'google_oauth2'
+      auth.save
+    end
```

Comment:
Maybe we can add another migration in the future that runs these updates again and then does a `Authentication.where(provider: ['facebooknewflow', 'googlenewflow']).delete_all` at the end. In a separate release, so the backend code is already updated and not using these old providers.